nvm.psd1

```diff
@@ -39,10 +39,10 @@
     # PowerShellHostVersion = ''
 
     # Minimum version of Microsoft .NET Framework required by this module
-    # DotNetFrameworkVersion = ''
+    DotNetFrameworkVersion = '4.7'
```

Comment:
The bug was only fixed in 4.7.1

Reply:
Are you sure it was fixed in 4.7.1 not 4.7?

Reply:
yes. it failed before AppVeyor deployed 4.7.1 and it also says so in the PowerShell docs:

> Fortunately, this has been fixed in .NET Framework 4.7.1 and in the Windows 10 Fall Creators Update. This version of the .NET Framework allows existing applications to "just work" without the need to modify and/or re-compile them. On these systems, a .NET Standard 2.0 based binary module will work in Windows PowerShell.
> 
> However, for Windows systems that have not been updated to .NET Framework 4.7.1 such a binary module will not run correctly in Windows PowerShell.

https://github.com/PowerShell/PowerShell/blob/master/docs/cmdlet-example/command-line-simple-example.md#using-a-net-standard-20-based-binary-module-in-windows-powershell